console/src/main/scripts/gulpfile.js

```diff
@@ -152,11 +160,11 @@ var gulpTsc = function(done) {
     });
 };
 
-gulp.task('tsc', ['clean-defs'], function (done) {
+gulp.task('tsc', ['clean-defs', 'tsd'], function (done) {
   gulpTsc(done);
 });
 
-gulp.task('tsc-live', ['copy-sources','clean-defs'], function (done) {
+gulp.task('tsc-live', ['copy-sources','clean-defs', 'tsd'], function (done) {
```

Comment:
Ouch, I have expected this to not be executed if there are no changes (the same as npm or bower). If that's not the case right now, it should be definitely fixed somehow.